generate_report.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+import os
+import json
+from datetime import datetime
+
+# Execute this file to generate a report of deployment latency.  Update max_version to get more recent releases and min_versions to go back further
+max_version = 50
+min_version = 10
+
+if os.path.exists("endpoints.json"):
+    print("Removing endpoints file")
+    os.remove("endpoints.json")
+
+f = open('output.csv', 'w')
```

Comment:
If you use `with open('output.csv', 'w') as output:` then you don't need to `close` below.